chemprop/uncertainty/predictor.py

```diff
@@ -123,17 +124,16 @@ def __call__(
         return
 
 
-@UncertaintyPredictorRegistry.register("conformal-quantile-regression")
-class ConformalQuantileRegressionPredictor(UncertaintyPredictor):
+@UncertaintyPredictorRegistry.register("quantile-regression")
+class QuantileRegressionPredictor(UncertaintyPredictor):
     def __call__(
         self, dataloader: DataLoader, models: Iterable[MPNN], trainer: pl.Trainer
     ) -> tuple[Tensor, Tensor]:
-        return
-
-
-@UncertaintyPredictorRegistry.register("conformal-regression")
-class ConformalRegressionPredictor(UncertaintyPredictor):
-    def __call__(
-        self, dataloader: DataLoader, models: Iterable[MPNN], trainer: pl.Trainer
-    ) -> tuple[Tensor, Tensor]:
-        return
+        individual_preds = []
+        for model in models:
+            predss = trainer.predict(model, dataloader)
+            individual_preds.append(torch.concat(predss, 0))
+        stacked_preds = torch.stack(individual_preds).float()
+        mean = stacked_preds[..., 0]
+        interval = torch.mean(stacked_preds[..., 1], dim=0)
+        return mean, interval
```

Comment:
Should this method return variance as the uncertainty to match the other methods (i.e., ensemble, dropout, MVE, evidential)?

Reply:
I'm not an expert, but it seems like we would have to assume a distribution to go from interval to variance, and I am not sure we can do that.

Reply:
I mean we train these things using max likelihood estimation from a unit normal gaussian, so it seems reasonable to assume a gaussian for this purpose

Reply:
The quantile loss function minimizes a weighted sum of absolute residuals, so I think it is not a maximum likelihood method. I will revert the change to still return the intervals as uncertainty.